Dockerfile

```diff
@@ -26,17 +26,17 @@ ENV VIRTUAL_ENV="${CISA_HOME}/.venv"
 # setuptools, and wheel) pre-venv because this Docker image is using Python
 # built from source and not a system Python package.
 RUN python3 -m pip install --no-cache-dir --upgrade \
-    pip==22.3.1 \
-    setuptools==65.7.0 \
+    pip==23.0 \
+    setuptools==67.3.1 \
     wheel==0.38.4 \
   && python3 -m pip install --no-cache-dir --upgrade \
-    pipenv==2022.12.19 \
+    pipenv==2023.2.4 \
   # Manually create Python virtual environment for the final image
   && python3 -m venv ${VIRTUAL_ENV} \
   # Ensure the core Python packages are installed in the virtual environment
   && ${VIRTUAL_ENV}/bin/python3 -m pip install --no-cache-dir --upgrade \
-    pip==22.3.1 \
-    setuptools==65.7.0 \
+    pip==23.0 \
+    setuptools==67.3.1 \
```

Comment:
Is it worth making variables for the `pip`, `pipenv`, `setuptools`, and `wheel` versions so that they can all be updated in one place?  This would also DRY out the code since the `pip`, `setuptools`, and `wheel` versions are currently being specified twice (lines 29-30 and 38-39).

Reply:
I support this idea.

Reply:
My original rationale was thinking that obfuscating the version might hinder any dependabot analysis that might automatically detect it as a dependency. After thinking about it though I thought that this approach is probably better overall to both ensure the same version is installed consistently and potentially avoidind multiple dependabot PRs around the frequently updating packages (`pip`, `pipenv`, and `setuptools`). These package versions can always be updated whenever dependabot generates a new `python` Docker image PR. Please see [685be83](https://github.com/cisagov/client-cert-update/pull/25/commits/685be832be4309a6f2ebd43faba87dac2a0ee68e).